src/Microsoft.Identity.Web.TokenAcquisition/TokenAcquisition.cs

```diff
@@ -747,6 +754,8 @@ private MergedOptions GetMergedOptions(string? authenticationScheme, TokenAcquis
                 mergedOptions = _tokenAcquisitionHost.GetOptions(authenticationScheme ?? tokenAcquisitionOptions?.AuthenticationOptionsName, out _);
             }
 
+            mergedOptions.IsTokenBinding = string.Equals(authenticationScheme, "MTLS_POP", StringComparison.OrdinalIgnoreCase);
```

Comment:
The authenticationScheme is the name of the configuration. And null is used by IdWeb for the default authentication scheme. See https://learn.microsoft.com/en-us/aspnet/core/security/authorization/limitingidentitybyscheme?view=aspnetcore-10.0

You can't assume that there is an authentication scheme for mTLS Pop (especially for outbound calls). The way to say it's mTLS Pop is the protocolScheme property of the AuthorizationHeaderOptions / DownstreamApiOptions